step/ftl_result_processor.go

```diff
@@ -0,0 +1,40 @@
+package step
+
+import (
+	"encoding/json"
+
+	toolresults "google.golang.org/api/toolresults/v1beta3"
+)
+
+func GetSuccessOfExecution(steps []*toolresults.Step) (bool, error) {
```

Comment:
Now this function and the tests are in the same package. You do not need to make it public anymore as private functions of a package can be accessed from the tests.

This will also solve your linting issue. 
(Failure on the CI: `/bitrise/go/src/github.com/bitrise-steplib/steps-virtual-device-testing-for-android/step/ftl_result_processor.go:9:1: exported function GetSuccessOfExecution should have comment or be unexported`)